README.md

```diff
@@ -57,6 +58,18 @@ sudo npm install -g svgo
 sudo apt-get install gifsicle
 ```
 
+Mozjpeg for Ubuntu:
+```bash
+sudo apt-get install autoconf automake libtool nasm make pkg-config git
```

Comment:
That's a lot of work to install this. Is there a way to install it with one simple `apt-get`?

Reply:
for ubuntu and debian, this is a deb package available, https://mozjpeg.codelove.de/binaries.html.